examples/kubernetes-config/kubernetes-containerd-tmpdir.json

```diff
@@ -2,7 +2,6 @@
   "apiVersion": "vlabs",
   "properties": {
     "orchestratorProfile": {
-      "orchestratorRelease": "1.17",
```

Comment:
got rid of unnecessary version declarations in these configs so we don't have to be annoyed by them in the future